### PR TITLE
update footer links for both donate site and thunderbird donate

### LINKED ISSUE #1018: [https://give.thunderbird.net/] Footer tweaks
Implementation ticket for footer changes.

**Impacted sites:**
[Thunderbird donate site](https://give.thunderbird.net/)

**Changes:**
1. Organized items by alphabetical order
2. Add "Press Center" copy. Link for [Press Center here](https://foundation.mozilla.org/press-center/)

image

---

**🗒  Design -> Dev Handoff Checklist**
(Feel free to remove items that are not applicable.)

- [x] Design has been finalized
- [x] Link to original design ticket: [#4868](https://github.com/mozilla/foundation.mozilla.org/issues/4868)
- [x] Link to InVision: [link here](https://mozilla.invisionapp.com/share/PJXWEC7BU86#/425165977_Footers-Thunderbird)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included

cc: @TheoChevalier - updates we will be doing to our English Footers

### LINKED ISSUE #1072: [https://donate.mozilla.org] Footer: add copy for CC license + link to Press Center Page
Implementation ticket for footer changes.

**Impacted sites:**
[Donate site](https://donate.mozilla.org)

**Changes:**
1. Removed "License" copy
2. Added Cookies as we have in our other web properties. [Link here](https://www.mozilla.org/privacy/websites/#cookies)
3. Removed "Donate" link since this is the Donate page
4. Organized items by alphabetical order
5. Added "Press Center" copy. Link for [Press Center here](https://foundation.mozilla.org/press-center/)
6. Added "Most content available under a Creative Commons license." Link for [Creative Commons license page is here](https://foundation.mozilla.org/about/website-licensing/)

image

---

**🗒  Design -> Dev Handoff Checklist**
(Feel free to remove items that are not applicable.)

- [x] Design has been finalized
- [x] Link to original design ticket [#4868](https://github.com/mozilla/foundation.mozilla.org/issues/4868)
- [x] Link to InVision. [Link here](https://mozilla.invisionapp.com/share/PJXWEC7BU86#/425165974_Footers-Donate).
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included

cc: @TheoChevalier - updates we will be doing to our English Footers

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #1018 
https://thunderbird-issue-1018--kzkmnp.herokuapp.com
Closes #1072 
https://donate-wagta-issue-1018-o0kkw6.herokuapp.com

### CODE REVIEW

**Review by KalobTaulien — APPROVED** [review]

**Review by taisdesouzalessa — CHANGES_REQUESTED** [review]
For the [donate site](https://donate-wagta-issue-1018-o0kkw6.herokuapp.com/en-US/) please replace "Cookie" by "Cookies".

Tks!

File: donate/templates/pages/base_page_master.html [hunk 1]
```diff
@@ -136,25 +136,25 @@ <h2 id="about-us-heading" class="footer-links__heading heading heading--bottom-m
                     <ul class="footer-links__group" aria-labelledby="about-us-heading">
                         {% block footer_links %}
                             <li class="footer-links__item">
-                                <a class="link link--dark-bg" href="http://donate.mozilla.org" target="_blank" rel="noopener">{% trans "Donate" %}</a>
+                                <a class="footer-links__link link link--dark-bg" href="https://www.mozilla.org/privacy/websites/#cookies" target="_blank" rel="noopener">{% trans "Cookie" %}</a>
```
Comment (mmmavis) [inline donate/templates/pages/base_page_master.html:139]:
```suggestion
                                <a class="footer-links__link link link--dark-bg" href="https://www.mozilla.org/privacy/websites/#cookies" target="_blank" rel="noopener">{% trans "Cookies" %}</a>
```

**Comment by mmmavis** [comment]
@taisdesouzalessa thanks for catching it! I've updated the PR!

**Review by taisdesouzalessa — APPROVED** [review]
yay! tks Mavis 💯 all good on the Design side!

**Pull request merged**